ote_sdk/ote_sdk/utils/segmentation_utils.py

```diff
@@ -184,27 +230,34 @@ def create_annotation_from_segmentation_map(
         # Contour retrieval mode CCOMP (Connected components) creates a two-level
         # hierarchy of contours
         contours, hierarchies = cv2.findContours(
-            label_index_map, cv2.RETR_CCOMP, cv2.CHAIN_APPROX_SIMPLE
+            label_index_map, cv2.RETR_CCOMP, cv2.CHAIN_APPROX_NONE
```

Comment:
Do we really need to disable the approximation? Seems it doesn't break the logic of subcontour selections but speeds up inference.